resources/repos.json

```diff
@@ -769,7 +769,7 @@
 		},
 		"dependencies": {
 			"Minecraft Version(s)": {
-				"1": "1.15.x, 1.16.x"
+				"1": "all"
```

Comment:
Help please, I tested it on Minecraft Beta 1.4 and it doesn't work!!!! pls fix

Reply:
argh